src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -82,23 +82,26 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       return;
     }
 
+    System.err.println(request.getRequestURI());
```

Comment:
nit: Remove these debug statements? Usually you should only print error on error paths in production code.

Reply:
Thanks, you're right, I missed those in my cleanup